Source/WebCore/dom/FullscreenManager.cpp

```diff
@@ -355,13 +347,10 @@ void FullscreenManager::exitFullscreen(CompletionHandler<void(ExceptionOr<void>)
             return completionHandler({ });
         }
 
-        // If there is a pending fullscreen element but no fullscreen element
-        // there is a pending task in requestFullscreenForElement(). Cause it to cancel and fire an error
-        // by clearing the pending fullscreen element.
+        // If there is no fullscreen element, cancel and fire an error
```

Comment:
Are we actually cancelling and firing an error here?

// If there is no fullscreen element, bail out early.

Reply:
I've addressed my own comment